.github/workflows/static-analysis.yml

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2018 - 2021 Settlers Freaks <sf-team at siedler25.org>
```

Comment:
we should always use `2005 - 2021` for our copyrights

Reply:
I'm not sure that is even legal... E.g. this file didn't exist in 2005 so we can't claim copyright for that time
For some files I checked the file-added date, but that is tedious so I got conservative with the start year